dump/src/lib.rs

```diff
@@ -227,14 +224,23 @@ impl DumpOptions {
         } else {
             (true, true)
         };
+        match self.format {
+            DumpFormat::Text => {
+                meta_dump(&mut to, meta, if no_limit { u32::MAX } else { width })?;
 
-        meta_dump(&mut to, meta, if no_limit { u32::MAX } else { width })?;
+                writeln!(to, "{:-<58}", "")?;
 
-        writeln!(to, "{:-<58}", "")?;
+                dump(&mut to, obj, width, 0, no_text_limit, no_limit)?;
 
-        dump(&mut to, obj, width, 0, no_text_limit, no_limit)?;
+                Ok(())
+            },
+            DumpFormat::Json => {
+                let json_obj = DicomJson::from(obj);
+                println!("{}", serde_json::to_string_pretty(&json_obj)?);
```

Comment:
`serde_json` has [`to_writer_pretty`](https://docs.rs/serde_json/latest/serde_json/fn.to_writer_pretty.html), which might be more efficient if we pass it [`stdout()`](https://doc.rust-lang.org/std/io/fn.stdout.html).